### DISCUSSION TITLE
ADNI clinical data

### DISCUSSION THREAD

**Discussion by AyaRabea** [aramis-lab/clinica#1341] [Q&A] [answered]
i need to use clinica to convert ADNI1 year t1 MRI images. i follow up the steps to download the MRI images, but in clinical data the new update of ADNI website made me unable to found were it.
can any one help me were can i find the clinical data
Thanks

**Comment by zzjkh** [comment]
metoo，I have also encountered this problem, the site has changed so much that the tutorials found are no longer applicable

**Comment by AliceJoubert** [comment, marked as answer]
Hi @zzjkh ,

Do you mind addressing that issue in our [google group](https://groups.google.com/g/clinica-user) ?

**Reply by zzjkh** [reply]
Of course ，I have already submitted the application. Thank you for your attention
